### DISCUSSION TITLE
How to setup the database with baseline?

### DISCUSSION THREAD

**Discussion by dtslvr** [prisma/prisma#10905] [Q&A]
I have a web app which is offered as a docker container. User are now running into the following error when trying to migrate the schema:

```
Error: P3005

The database schema for `postgres:5432` is not empty. Read more about how to baseline an existing production database: https://pris.ly/d/migrate-baseline
```

There is a [script](https://github.com/ghostfolio/ghostfolio/blob/main/package.json#L25) to setup the database (push and seed). Is there an easy way to also baseline the database with the latest migration to be prepared for future migrations?

Thanks in advance
Thomas

**Comment by janpio** [comment]
Which command gives this error message?

**Reply by dtslvr** [reply]
> Which command gives this error message?

`prisma migrate deploy`
